testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/BuildConfigurationTestBase.java

```diff
@@ -144,35 +143,34 @@ private static File hackFixHostConfig(File hostConfigFile, String hostName, Stri
         return file.toFile();
     }
 
-    static File hackFixDomainConfig(File hostConfigFile) {
+    static File hackFixDomainConfig(File domainConfigFile) {
         final File file;
-        final BufferedWriter writer;
+
         try {
-            file = File.createTempFile("domain", ".xml", hostConfigFile.getAbsoluteFile().getParentFile());
+            file = File.createTempFile("domain", ".xml", domainConfigFile.getAbsoluteFile().getParentFile());
             file.deleteOnExit();
-            writer = new BufferedWriter(new FileWriter(file));
+
         } catch (IOException e) {
             throw new RuntimeException(e);
         }
-        try {
-            BufferedReader reader = new BufferedReader(new FileReader(hostConfigFile));
-            try {
-                String line = reader.readLine();
-                while (line != null) {
-                    int start = line.indexOf("java.net.preferIPv4Stack");
-                    if (start < 0) {
-                        writer.write(line);
-                    }
-                    writer.write("\n");
-                    line = reader.readLine();
+        try (BufferedReader reader = new BufferedReader(new FileReader(domainConfigFile));
+             final BufferedWriter writer = Files.newBufferedWriter(file.toPath())
+        ) {
+            String line = reader.readLine();
+            while (line != null) {
+                if (line.contains("<security-setting name=\"#\">")) { //super duper hackish, just IO optimization
+                    writer.write("        <journal file-size=\"1024\" />");
```

Comment:
Ugh. But ok I guess.

Reply:
without this change, we by default on boot create 24mb of Artemis journal files. 
with this change this goes down to ~2mb